pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroup.java

```diff
@@ -321,6 +321,12 @@ protected BytesAndMessagesCount getGlobalUsageStats(ResourceGroupMonitoringClass
 
     protected BytesAndMessagesCount updateLocalQuota(ResourceGroupMonitoringClass monClass,
                                                      BytesAndMessagesCount newQuota) throws PulsarAdminException {
+        // Only the Publish side is functional now; add the Dispatch side code when the consume side is ready.
```

Comment:
Will this check go away when the consume side is ready?

Reply:
Yes.

Reply:
@kaushik-develop , May I check if the consume side is ready or not now? Because we can still see this part in the master branch and it seems not to work when I set the `--msg-dispatch-rate 10 --byte-dispatch-rate 2000` to a resourcegroup. Can you advise? 
Thanks,
Hongten